database/migrations/20170922112122_create_event_log_table.js

```diff
@@ -0,0 +1,31 @@
+const databaseUtils = require('../utils')
+
+exports.up = knex =>
+  Promise.all([
+    knex.schema.createTable('event_logs', table => {
+      table.timestamp('occurred_at').defaultTo(knex.fn.now())
+      table.string('user_id').notNullable()
+      table.string('type').notNullable()
+      table.jsonb('metadata').notNullable()
+    })
+    .then(tables =>
+      databaseUtils.migrateCheckLogs(knex)
```

Comment:
we do not need these to be in utils going forward. just define these functions at the bottom of this migration and call them here.